src/blueapi/utils/ophyd_async_connect.py

```diff
@@ -45,3 +39,14 @@ async def _wait_for_tasks(tasks: Dict[asyncio.Task, str], timeout: float):
             logging.exception(f"  {tasks[t]}:", exc_info=t.exception())
     if pending or raised:
         raise NotConnected("Not all Devices connected")
+
+
+def format_error_message(tasks: Dict[asyncio.Task, str], t: asyncio.Task) -> str:
```

Comment:
I think this should have a more descriptive name or be prefixed with an `_`

Reply:
doing the `_` option now